functions/streamtoidv/function.dev.json

```diff
@@ -3,8 +3,13 @@
   "handler": "main.handle",
   "description": "The change integration stream consumer that updates the identity vault",
   "role": "arn:aws:iam::656532927350:role/CISLambdaFunctionRoles-StreamToIDV-T5IH2261NJZM",
+  "environment": {
+    "CIS_ARN_MASTER_KEY": "arn:aws:kms:us-west-2:656532927350:key/9e231aa0-04e4-4517-a45d-633c3bb055f0",
```

Comment:
probs... but what ya gonna do.  I'll answer the bounty ticket when it comes in ;)